### DISCUSSION TITLE
Why an additional package for mobile?

### DISCUSSION THREAD

**Discussion by Saidnazar08** [ant-design/ant-design#45424] [General]
Can someone explain to me why there are two libraries, antd and antd mobile? Wouldn't it be easier for us and maintainers to develop just one responsive component rather than one for desktop and one for mobile? I'm asking because I wanted to use the date range picker and tour components, but they aren't responsive.

**Comment by zombieJ** [comment]
There have some difference between PC & mobile version. In simple words is that PC version provides more ability but which mean bundle size is larger. And mobile version is much simplify which fully excludes the PC interactive logic and the bundle size is much smaller.

For PC version. Yes, it should to do the responsive for the mobile env. It's in the plan but not fast since we already has mobile version.

**Reply by Saidnazar08** [reply]
Thank you for the clarification.
